components/partials/ActionLink.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <a :class="[colorItem]" href="#">
+  <span class="tekstStyl">
+  {{ linktext }}
+  </span>
+  <span :class="[iconClass, colorIcon]" class="ikonaStyl"></span>
+  </a>
+</template>
+
+<script>
+export default {
+  props: {
+    linktext: {
+      type: String,
+      default: 'puste'
+    },
+    iconClass: {
+      type: String
+    },
+    colorIcon: {
+      type: String
+    },
+    colorItem: {
+      type: String
+    }
+  }
+}
+</script>
+
+<style lang="scss">
+.firstIconColor {
+  background-color: #494949;
+  color: #F3C000;
+  align-self: flex-start;
+
+  &:before {
+    font-size: 4vw;
+  }
+}
+
```

Comment:
I would rather use classes with color variants applied to link rather than sending names of classes through props